client/src/containers/ResetPassword/index.tsx

```diff
@@ -0,0 +1 @@
+export { default } from "./ResetPassword";
```

Comment:
I don't catch this notation

Reply:
Just compare importing components with or without this statement and index.tsx file

`import SignUp from "./containers/SignUp/SignUp";
import ResetPassword from "./containers/ResetPassword";`